orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -202,7 +206,7 @@ def OnChange(changeType, level, resource) -> None:  # noqa: ARG001
 
     if changeType == orthanc.ChangeType.STABLE_STUDY and ShouldAutoRoute():
         print("Stable study: %s" % resource)  # noqa: T201
-        SendViaStow(resource)
+        SendViaFTPS(resource)
```

Comment:
should we delete the SendViaStow function as it's unused?

Reply:
I would say yes, but maybe it will become relevant again at some piont? @stefpiatek?

Reply:
yeah we want to keep that functionality when we start stopping this from being hardcoded and make the endpoint we send to configurable